examples/mock-date/README.md

```diff
@@ -1 +1,9 @@
-#How to mock data
+# How To Mock Date
+
+**Test Code**: [index.js](index.js)
+
+[`mockdate`](https://www.npmjs.com/package/mockdate) is an npm module that allows you to substitute the current date.
+
+Use the `clientScripts` function to inject the `mockdate` module and add the code that uses this module to set the browser date value with `MockDate.set`, passing a string with the date as its argument.
```

Comment:
This sentence does not make sense. What are you trying to say here?

Reply:
@dirk-pieterse the function injects (introduces) the npm module we've mentioned before and then we add the code that alters the browser date value. We use the `mockdate` module to do that. The function `MockDate.set` takes a new date as an argument. This allows substituting the current date with another date.

Reply:
'passing a string with the date as its argument' passing it to what?